backend/src/routes/account.js

```diff
@@ -16,6 +14,20 @@ router.get('/',
     res.send(req.user)
   })
 
+router.patch('/', async (req, res, next) => {
+  if (!req.user) return
```

Comment:
we should `return next({ status: 400 })`

Reply:
done